src/main/java/hudson/plugins/robot/tokens/RobotFailTokenMacro.java

```diff
@@ -20,6 +22,11 @@
 	public String evaluate(AbstractBuild<?, ?> context, TaskListener listener,
 			String macroName) throws MacroEvaluationException, IOException,
 			InterruptedException {
+		return evaluate(context, context.getWorkspace(), listener, macroName);
+	}
+
+	@Override
```

Comment:
Please add a comment to the url https://github.com/jenkinsci/email-ext-plugin/blob/master/src/main/java/hudson/plugins/emailext/plugins/content/FailedTestsContent.java so that why we have two forms of `evaluate()` is explained.